src/main/java/com/shredder/example/aws/s3/S3Helper.java

```diff
@@ -166,6 +211,16 @@ public void uploadItemToBucket(String bucketName, String keyOrPath, String newFi
         }
     }
 
+    /**
+     * This function downloads the file from the bucket. If the path is empty then it will print invalid file
+     * storage directory.
+     * @param bucketName aditya123
+     * @param keyOrPath aditya123/folder1/
```

Comment:
key cannot/should not contain the bucket name again

Reply:
Bucket name removed from key/path